AllReadyApp/Web-App/AllReady/Features/Resource/ResourcesByCategoryQueryHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using AllReady.Models;
+using MediatR;
+
+namespace AllReady.Features.Resource
+{
+    public class ResourcesByCategoryQueryHandler : IRequestHandler<ResourcesByCategoryQuery, List<Models.Resource>>
+    {
+        private readonly IAllReadyDataAccess dataAccess;
+
+        public ResourcesByCategoryQueryHandler(IAllReadyDataAccess dataAccess)
+        {
+            this.dataAccess = dataAccess;
+        }
+
+        public List<Models.Resource> Handle(ResourcesByCategoryQuery message)
+        {
+            return dataAccess.GetResourcesByCategory(message.Category).Distinct().ToList();
```

Comment:
Why do you need `.Distinct()` here? Each `Resource` will be its own reference so by nature they will be distinct.